x/feemarket/keeper/eip1559.go

```diff
@@ -39,7 +39,7 @@ func (k Keeper) CalculateBaseFee(ctx sdk.Context) *big.Int {
 		return nil
 	}
 
-	parentGasUsed := k.GetBlockGasUsed(ctx)
+	parentGasUsed := k.GetBlockGasWanted(ctx)
```

Comment:
We need to take into consideration the `MinGasMultiplier`, some chains may want to set the multiplier to zero to keep previous behavior, in that case, the cost of setting a high gas limit is zero.

Reply:
so you want to revert the changes to rely on `GasUsed` instead?

Reply:
I think we should move from `gasUsed` to `max(gasLimit * multiplier, gasUsed)`, rather than simply `gasLimit`?

Reply:
> so you want to revert the changes to rely on `GasUsed` instead?

No, but it'd be good to keep it compatible with that case, I can foresee after abci++ when we include tx by gas used, people may want to set the `multipler` to zero to revert back to old approach.

Reply:
Do you mean to `1` instead of `0`?